packages/catalog-process/src/services/catalogService.ts

```diff
@@ -438,6 +438,9 @@ export function catalogServiceBuilder(
         riskAnalysis: [],
         isSignalHubEnabled: seed.isSignalHubEnabled,
         isDelegable: seed.isDelegable,
+        isClientAccessDelegable: seed.isDelegable
+          ? seed.isClientAccessDelegable
+          : undefined,
```

Comment:
Until we clarify what to do with this flag's edit, I would keep it editable in `updateEservice`, which is the update when the service is still in draft, as we do for `isDelegable`. Otherwise, we are allowing an inconsistent scenario: this flag remains true even though the other one is updated to false, which shall not be possible. Once we clarify what to do on edit, we will adjust (see dedicated task: https://pagopa.atlassian.net/browse/PIN-5700)